analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MarkWindowsFormsMainWithStaThread.vb

```diff
@@ -14,8 +14,6 @@ Namespace Tests.TestCases
   Public Class Program_01
     Shared Sub Main() ' Noncompliant {{Add the 'STAThread' attribute to this entry point.}}
 '              ^^^^
-      Dim winForm As Form = New Form
```

Comment:
why remove this?

Reply:
or is it just looking at the namespace?

Reply:
After some offline talk with Andrei, we decided there is no action to be taken.